extensions/omni_vfs/src/file_info_table_fs.sql

```diff
@@ -15,7 +23,7 @@ select
 from
     table_fs_files                f
     inner join match              m on f.id = m.id
-    inner join table_fs_file_data d on m.id = d.file_id
+    left join table_fs_file_data  d on m.id = d.file_id
```

Comment:
Do I get this correctly that you're changing this to a left join to accommodate for the fact that the root entry does not / might not exist? If so, do you think it'd be worthwhile to explain this briefly in a comment?